src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPaginatedItems(items: string[], perPage: number) {
+  const paginated = [];
```

Comment:
Similar to the 'getNumbers' function, you should also initialize 'paginated' with a specific type, such as 'string[][]', to ensure the array contains arrays of strings.

Reply:
fixed